test/cli/inline-suppress_test.py

```diff
@@ -26,7 +26,7 @@ def __create_unused_function_compile_commands(tmpdir):
     return compile_commands
 
 
-def test1():
+def test_1():
```

Comment:
Nothing to do with private but closer to the pytest convention (they were detected and executed - so no functional change) so you can find all the functions by a simple search.

Reply:
okay